integration-tests/sanity/sanity_test.go

```diff
@@ -134,7 +134,7 @@ var _ = Describe("Sanity", func() {
 		It("should delete cstor pvc", func() {
 			time.Sleep(waitTime * time.Second)
 			// Getting the PVC artifact
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.CStorPVCArtifacts)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.CStorPVCArtifacts)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added